qtpy/tests/test_main.py

```diff
@@ -140,3 +168,16 @@ def test_qt_api_environ(api):
     raise AssertionError('QtPy imported despite bad QT_API')
 """
     subprocess.check_call([sys.executable, "-Oc", cmd], env=env)
+
+
+def test_qt_version():
+    """
+    If QT_VERSION is specified, check that one of the correct Qt wrappers was used
+    """
+
+    QT_VERSION = os.environ.get("QT_VERSION", "").lower()
+
+    if QT_VERSION == "qt5":
+        assert_qt5()
+    elif QT_VERSION == "qt6":
+        assert_qt6()
```

Comment:
Not totally sure but, I think we are not setting the `QT_VERSION` env var over the CI so for the moment this test is passing without doing any assert?

Reply:
Yes, I don't think `QT_VERSION` is set in CI, ideally both possibilities should be checked. Do you happen to know how this works for the similar test [`test_qt_api`](https://github.com/Bzero/qtpy/blob/b8c9d20e508b66d269a250ad22f9b7c9c64ca139/qtpy/tests/test_main.py#L97)?

Reply:
I think currently the `test_qt_api` is going through the `else` branch 😅 Checking the script that does the test setup ([test.sh](https://github.com/Bzero/qtpy/blob/b8c9d20e508b66d269a250ad22f9b7c9c64ca139/.github/workflows/test.sh)) I'm not seeing us setting up `QT_API` 🤔 Maybe we should try to check a way to setup `QT_API` (and `QT_VERSION`) for the tests (I think there is a `USE_QT_API` that can be set for test but I think we are not using it here either). What do you think @ccordoba12 ?

Reply:
I also think that's a good idea but I don't understand very well how to set up those env vars in our CIs.